pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+line-length = 130
+
+[tool.flake8]
+max-line-length = 130
+
+[tool.isort]
+profile = "black"
+line_length = 130
```

Comment:
These line lengths don't match the values in `setup.cfg` or the existing action. It's also not configured to ignore the warnings we're ignoring. Do we need this file?

Reply:
pyproject.toml seems to fulfill the same purpose of the `setup.cfg` so i will look for a way to use that configuration file instead.